build-system/build.conf.js

```diff
@@ -28,7 +28,7 @@ const defaultPlugins = [
 ];
 
 module.exports = {
-  plugins: ({isEsmBuild, isCommonJsModule, isForTesting}) => {
+  plugins({isEsmBuild, isCommonJsModule, isForTesting}) {
```

Comment:
`isCommonJsModule` was something I added during a previous closure compiler upgrade as a workaround to some errors I was seeing. (See https://github.com/ampproject/amphtml/commit/e4ea57103030da7e7959dba3eded96855e47af03.) I wonder if it can be removed now.

Call site:
https://github.com/ampproject/amphtml/blob/c4a663d0ba13d0488c6fe73c55dc8c971ac6ec0d/build-system/build.conf.js#L51-L55